code/modules/mob/living/carbon/xenomorph/castes/spiderling/spiderling.dm

```diff
@@ -65,10 +64,11 @@
 		go_to_obj_target(source, A)
 		return
 
-/// Sets escorted atom to our pre-defined default escorted atom, which by default is this spiderling's widow
+/// Sets escorted atom to our pre-defined default escorted atom, which by default is this spiderling's widow, and commands the spiderling to follow it
 /datum/ai_behavior/spiderling/proc/only_set_escorted_atom(source, atom/A)
```

Comment:
Could you rename this proc to be more descriptive? Something like `revert_to_default_escort`. Also the `atom/A` param is unused so delete that as well.